<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDaoYouGmUi</class>
 <widget class="QWidget" name="CDaoYouGmUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineItemName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>17</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>回车将搜索TID并拷贝到剪切板</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TID</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textGm">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCopy">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCopyName">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textItemName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineItemTid">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>117</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineItemName</tabstop>
  <tabstop>textGm</tabstop>
  <tabstop>btnCopy</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
